### PR TITLE
PMM-3909 Adding support for PMM2 Custom Client installation

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -256,94 +256,103 @@ pipeline {
...
+                                        export BUILD_ID=dear-jenkins-please-dont-kill-virtualbox
+                                        export JENKINS_NODE_COOKIE=dear-jenkins-please-dont-kill-virtualbox
+                                        export JENKINS_SERVER_COOKIE=dear-jenkins-please-dont-kill-virtualbox
+                                        tar -zxpf pmm2-client.tar.gz
+                                        rm -r pmm2-client.tar.gz
+                                        export PMM_CLIENT_BASEDIR=\\\$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+                                        export PATH="$PWD/pmm2-client-2.0.0/bin:$PATH"
+                                        echo "export PATH=$PWD/pmm2-client-2.0.0/bin:$PATH" >> ~/.bash_profile
+                                        source ~/.bash_profile
+                                        touch pmm-agent.log
```
Comment (hors) [inline pmm/aws-staging-start.groovy]: I think it should be automatically created in line 350 (we need to check it).

File: pmm/aws-staging-start.groovy [hunk 2]
```diff
@@ -256,94 +256,103 @@ pipeline {
...
+                                        pmm-admin --version
+                                        pmm-agent setup --config-file=$PWD/pmm2-client-2.0.0/config/pmm-agent.yaml --server-insecure-tls --server-address=\$IP:443
+                                        sleep 10
+                                        JENKINS_NODE_COOKIE=dontKillMe nohup bash -c 'pmm-agent --config-file=$PWD/pmm2-client-2.0.0/config/pmm-agent.yaml >> pmm-agent.log 2>&1 &'
```
Comment (hors) [inline pmm/aws-staging-start.groovy]: pmm-agent.yaml > pmm-agent.log 2>&1 &

File: pmm/aws-staging-start.groovy [hunk 3]
```diff
@@ -256,94 +256,103 @@ pipeline {
...
+                                        fi
+                                        export BUILD_ID=dear-jenkins-please-dont-kill-virtualbox
+                                        export JENKINS_NODE_COOKIE=dear-jenkins-please-dont-kill-virtualbox
+                                        export JENKINS_SERVER_COOKIE=dear-jenkins-please-dont-kill-virtualbox
+                                        tar -zxpf pmm2-client.tar.gz
+                                        rm -r pmm2-client.tar.gz
+                                        export PMM_CLIENT_BASEDIR=\\\$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+                                        export PATH="$PWD/pmm2-client-2.0.0/bin:$PATH"
+                                        echo "export PATH=$PWD/pmm2-client-2.0.0/bin:$PATH" >> ~/.bash_profile
+                                        source ~/.bash_profile
```
Comment (hors) [inline pmm/aws-staging-start.groovy]: Is it useful ?

**Comment by puneet0191** [comment]
Fixed, comments

**Pull request merged**